### DISCUSSION TITLE
Consider implementing Hillobar's subsampling algorithm to get inswapper above 128 output.  256/512 and potentially even 1024

### DISCUSSION THREAD

**Discussion by azn8573** [hacksider/Deep-Live-Cam#323] [Ideas]
Hilliobar implemented a subsampling algorithm with good results in the recent Rope Pearl update.

Can reference this post to see: https://www.reddit.com/r/StableDiffusion/comments/1d21s4t/comment/l61q17x/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by vlednerk** [comment]
@hacksider yo u should try use da shut 
Ps I'll tag u , so should say we r awaiting for vicam 😅
